SYCL/Plugin/level_zero_device_free_mem.cpp

```diff
@@ -0,0 +1,31 @@
+// REQUIRES: level_zero, level_zero_dev_kit
+
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %level_zero_options %s -o %t.out
+// RUN: env ZES_ENABLE_SYSMAN=1 %GPU_RUN_PLACEHOLDER %t.out 2>&1 %GPU_CHECK_PLACEHOLDER
+//
+// The test is to check that the free device memory is reported be Level Zero
+// backend
+//
+// CHECK: Free device memory

```

Comment:
CHECK will pass only if ext_intel_device_info_free_memory is supported by the device. I think we expect the test to pass even if device doesn't support it. The only option I see is to remove this CHECK, I guess we don't have dynamic checks.

Reply:
But we expect that ALL Level Zero devices support this extension.

Reply:
@againull : are you OK with the test? we need this to complete the new feature (the compiler fix was merged)

Reply:
Yes, I am ok with the test, thanks for clarification.